generators/python/src/fern_python/generators/sdk/declaration_referencers/environments_enum_declaration_referencer.py

```diff
@@ -8,11 +8,11 @@ def __init__(self, client_class_name: str, skip_resources_module: bool):
         super().__init__(skip_resources_module=skip_resources_module)
         self._client_class_name = client_class_name
 
-    def get_filepath(self, *, name: None) -> Filepath:
+    def get_filepath(self, *, name: None, as_request: bool = False) -> Filepath:
         return Filepath(
             directories=(),
             file=Filepath.FilepathPart(module_name="environment"),
         )
 
-    def get_class_name(self, *, name: None) -> str:
+    def get_class_name(self, *, name: None, as_request: bool = False) -> str:
```

Comment:
My preference would be to update every call site so that you know you are not accidentally passing in the wrong value anywhere

Reply:
How do you mean?

Reply:
b/c we default `as_request: bool = False` it means that we won't have to update every callsite. If we did `as_request: bool` every function invocation would have to make sure it passes stuff in correctly. 

It's less convenient

Reply:
Ah I see, yeah I can just not default the value
It's just a bigger diff is the only reason I defaulted